#8 Waffle badge

nejdaw opened the pull request:
This commit closes #8 .
Merge this to receive a badge indicating the number of issues in the ready column on your waffle.io board at https://waffle.io/Viridovix/show-me-the-code-github

mkordas commented:
@dawidnejman 
Commit message still does not fit our [commit naming convention](https://github.com/Viridovix/show-me-the-code-github/blob/master/CONTRIBUTING.md#commit-naming-convention). If you'd like to make rules less strict, let's propose PR for it first.

nejdaw committed:
#8 Add waffle.io badge

This commit closes #8.
Merge this to receive a badge indicating the number of issues in the ready column on your waffle.io board at https://waffle.io/Viridovix/show-me-the-code-github

nejdaw commented:
@mkordas forgot about a verb in present tense. Good catch
